helm/chaos-mesh/templates/controller-manager-rbac.yaml

```diff
@@ -57,9 +57,6 @@ rules:
       - watch
       - list
       - get
-  - apiGroups: [ "" ]
-    resources: [ "configmaps" ]
-    verbs: [ "*" ]
```

Comment:
Is there any other problem with changing from `*` to `"get", "list", "watch", "delete", "update", "patch"`?

Reply:
`create` is also required, so it's basically the same as '*'